source/install/self-managed-changelog.md

```diff
@@ -3,13 +3,115 @@
+ - Fixed an issue where Direct Message notifications were missing the sender name with Collapsed Reply Threads enabled.
+ - Fixed an issue where keyboard shortcuts were not working with Global Threads.
+ - Fixed an issue where API allowed changing the name of the Town Square channel.
+ - Fixed an issue where errors were logged if a user disabled notifications.
+ - Fixed an issue where a channel was not immediately removed from the sidebar when the current user was removed from it.
+ - Fixed a potential server crash when creating or updating posts with permalink previews.
+ - Fixed an issue where permalinks created from saved posts did not correctly redirect to the correct team.
+ - Fixed an issue where long file extension names pushed out of the bounds of the module.
+   
+### config.json
+Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``ServiceSettings`` in ``config.json``:
+   - Added a new config setting ``DeveloperFlags``.
+ - Removed ``DesktopLatestVersion`` and ``DesktopMinVersion`` config settings.
+
+### API Changes
+ - Added a new ``IsEnterpriseReady()`` plugin API.
+ - Added a new ``GET /api/v4/roles`` API endpoint.
+ - Added new ``UpdateCustomStatus`` and ``RemoveUserCustomStatus`` plugin APIs for user custom status.
+ - Added CRUD methods for user sessions to the plugin API.
+ 
+### Go Version
+ - v6.2 is built with Go ``v1.16.7``.
+
+### Known Issues
+ - Member type is missing from autocomplete [MM-38989](https://mattermost.atlassian.net/browse/MM-38989).
+ - File upload might fail for SVG files [MM-38982](https://mattermost.atlassian.net/browse/MM-38982).
+ - ``CTRL/CMD + SHIFT + A`` shortcut does not open **Settings** [MM-38236](https://mattermost.atlassian.net/browse/MM-38236).
+ - Known issues related to the Collapsed Reply Threads (Beta) are [listed here](https://docs.mattermost.com/messaging/organizing-conversations.html#known-issues).
+ - Adding an at-mention at the start of a post draft and pressing the leftwards or rightwards arrow can clear the post draft and the undo history [MM-33823](https://mattermost.atlassian.net/browse/MM-33823).
+ - Google login fails on the Classic mobile apps.
+ - Status may sometimes get stuck as **Away** or **Offline** in High Availability mode with IP Hash turned off.
+ - Searching stop words in quotation marks with Elasticsearch enabled returns more than just the searched terms.
+ - The team sidebar on the desktop app does not update when channels have been read on mobile.
+ - Slack import through the CLI fails if email notifications are enabled.
+ - Push notifications don't always clear on iOS when running Mattermost in High Availability mode.
+
```

Comment:
- Boards are not refreshing on creation. See [GitHub discussion](https://github.com/mattermost/focalboard/discussions/1971) for more information.